src/Algebra/Structures.agda

```diff
@@ -21,7 +21,7 @@ module Algebra.Structures
 -- The file is divided into sections depending on the arities of the
 -- components of the algebraic structure.
 
-open import Algebra.Core
+open import Algebra.Core using (Op₁; Op₂)
```

Comment:
This is (semantically) redundant, as `Algebra.Core` consists of only those definitions... but was that your intention here, to draw attention to that?

I'm conscious that the library has several/many such small leaf modules, for the sake of resolving some dependency knots, so not clear about the value-add of this patch?

Reply:
I 'mechanically' put in `using` without much regard to what's in the downstream file, as a means to document what is being used in the current file, i.e. establish an explicit vocabulary / domain of discourse. That it turns out to be the whole file *right now* doesn't mean that `Algebra.Core` won't grow in the future! So these `using` future-proof things from accidental namespace pollution.

The value-add of this particular patch is tiny indeed. I still prefer my imports to be explicit, so that I don't have to keep in active memory all that's in those modules.